pagai/json_encoder.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+import decimal
+import flask
+
+
+class MyJSONEncoder(flask.json.JSONEncoder):
+    """
+    We make our JSONEncoder to override the default method.
+    Decimal values are not handled by Flask's JSONEncoder so
```

Comment:
Hum, I know it's copied from fhir-river but maybe we should speak about all the case we handle or none of them, what do you think?

Reply:
agreed, done.